components/calendar/benches/date.rs

```diff
@@ -258,7 +258,7 @@ fn date_benches(c: &mut Criterion) {
         &fxs,
         icu::calendar::islamic::IslamicUmmAlQura::new_always_calculating(),
         |y, m, d| {
-            Date::try_new_ummalqura_date(
+            Date::try_new_ummalqura_date_with_calendar(
```

Comment:
shouldn't it be `try_new_ummalqura_with_calendar`? We don't say `DecimalFormatter::try_new_decimal_formatter_with_any_provider`

Reply:
good catch! can't believe I didn't notice this